src/style.css

```diff
@@ -1,3 +1,78 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
 body {
+  background-color: #858585;
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  font-size: 15px;
+  align-items: center;
+  height: 60px;
+  text-decoration: none;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  background-color: #fff;
+}
+
+.nav__link {
+  text-decoration: none;
```

Comment:
Don't use tag names for styling. Here you're styling the 'a' tag directly. It's better to style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`.